no change     /curc/sw/install/miniforge3/24.11.3-0/condabin/conda
no change     /curc/sw/install/miniforge3/24.11.3-0/bin/conda
no change     /curc/sw/install/miniforge3/24.11.3-0/bin/conda-env
no change     /curc/sw/install/miniforge3/24.11.3-0/bin/activate
no change     /curc/sw/install/miniforge3/24.11.3-0/bin/deactivate
no change     /curc/sw/install/miniforge3/24.11.3-0/etc/profile.d/conda.sh
no change     /curc/sw/install/miniforge3/24.11.3-0/etc/fish/conf.d/conda.fish
no change     /curc/sw/install/miniforge3/24.11.3-0/shell/condabin/Conda.psm1
no change     /curc/sw/install/miniforge3/24.11.3-0/shell/condabin/conda-hook.ps1
no change     /curc/sw/install/miniforge3/24.11.3-0/lib/python3.12/site-packages/xontrib/conda.xsh
no change     /curc/sw/install/miniforge3/24.11.3-0/etc/profile.d/conda.csh
no change     /home/mlippincott@xsede.org/.bashrc
No action taken.
[NbConvertApp] Converting notebook notebooks/colocalization_class.ipynb to script
[NbConvertApp] Writing 3420 bytes to scripts/colocalization_class.py
[NbConvertApp] Converting notebook notebooks/featurization.ipynb to script
[NbConvertApp] Writing 10869 bytes to scripts/featurization.py
[NbConvertApp] Converting notebook notebooks/merge_outputs.ipynb to script
[NbConvertApp] Writing 687 bytes to scripts/merge_outputs.py
[NbConvertApp] Converting notebook notebooks/run_featurization.ipynb to script
[NbConvertApp] Writing 9724 bytes to scripts/run_featurization.py
Input Notebook:  run_featurization.ipynb
Output Notebook: run_featurization.ipynb
Executing:   0%|          | 0/16 [00:00<?, ?cell/s]Executing notebook with kernel: python3
Executing:   6%|▋         | 1/16 [00:09<02:15,  9.02s/cell]Executing:  50%|█████     | 8/16 [00:17<00:15,  1.95s/cell]Executing:  81%|████████▏ | 13/16 [24:28<07:01, 140.63s/cell]Executing:  81%|████████▏ | 13/16 [24:29<05:39, 113.01s/cell]
Traceback (most recent call last):
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/bin/papermill", line 8, in <module>
    sys.exit(papermill())
             ^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/click/core.py", line 1161, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/click/core.py", line 1082, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/click/core.py", line 1443, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/click/core.py", line 788, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/papermill/cli.py", line 235, in papermill
    execute_notebook(
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/papermill/execute.py", line 131, in execute_notebook
    raise_for_execution_errors(nb, output_path)
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_featurization/lib/python3.12/site-packages/papermill/execute.py", line 251, in raise_for_execution_errors
    raise error
papermill.exceptions.PapermillExecutionError: 
---------------------------------------------------------------------------
Exception encountered at "In [8]":
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[8], line 135
    133 del object_loader
    134 del size_shape_dict
--> 135 del object_measurements
    136 del output_dict
    137 del neighbors_out_dict

NameError: name 'object_measurements' is not defined

Featurization done
